dune

```diff
@@ -6,14 +6,24 @@
           -safe-string        ;; Immutable strings.
           -annot              ;; Dumps information per (file) module about types, bindings, tail-calls, etc. Used by some tools.
           -warn-error -a      ;; Do not treat warnings as errors.
-          -w A-4-44-45-48-60-67  ;; Ignores warnings 4, 44, 45, 48, 60, and 67.
+          -w A-4-42-44-45-48-60-67  ;; Ignores warnings 4, 42, 44, 45, 48, 60, and 67.
+          -g                  ;; Adds debugging information to the resulting executable / library.
+ )))
+(nowarnings
+  (flags (:standard
+          -strict-formats     ;; Disallows legacy formats.
+          -strict-sequence    ;; Enforces the lhs of a sequence to have type `unit'.
+          -safe-string        ;; Immutable strings.
+          -annot              ;; Dumps information per (file) module about types, bindings, tail-calls, etc. Used by some tools.
+          -warn-error -a      ;; Do not treat warnings as errors.
+          -w -A               ;; Ignore all warnings
```

Comment:
What's going on here?

Reply:
The build profile `nowarnings` should be deleted unless we elect to keep such a profile. Personally, I don't think we should keep it.

Reply:
I added it for the purposes of the refactoring, but I don't see a point in keeping it

Reply:
I guess it might be useful, but it's pretty easy to add back in if anyone wants to do this again. I'll remove it for now.